test/UnitTests/MSTestAdapter.UnitTests/Execution/TestClassInfoTests.cs

```diff
@@ -207,7 +207,7 @@ public void RunClassInitializeShouldThrowIfTestContextIsNull()
         TestResult result = GetResultOrRunClassInitialize(null);
         var exception = result.TestFailureException as TestFailedException;
         Verify(exception is not null);
-        Verify(result.Outcome == UTF.UnitTestOutcome.Failed);
+        Verify(result.Outcome == UTF.UnitTestOutcome.Error);
```

Comment:
It's hard to understand if the change here makes sense or not so I'll approve but please have a quick look.

Reply:
The unit test passes `null` as TestContext which is something that never happens in production actually. Even if it happens, I think `Error` makes better sense as it's more of MSTest error rather than a "failing" test on user side.